spyder/plugins/editor/widgets/base.py

```diff
@@ -563,39 +525,32 @@ def select_current_cell_in_visible_portion(self):
 
         return cursor, cell_full_file
 
-
     def go_to_next_cell(self):
         """Go to the next cell of lines"""
         cursor = self.textCursor()
-        cursor.movePosition(QTextCursor.NextBlock)
-        cur_pos = prev_pos = cursor.position()
-        while not self.is_cell_separator(cursor):
-            # Moving to the next code cell
-            cursor.movePosition(QTextCursor.NextBlock)
-            prev_pos = cur_pos
-            cur_pos = cursor.position()
-            if cur_pos == prev_pos:
-                return
+        block = cursor.block()
+        try:
+            footer = next(document_cells(

```

Comment:
Is this to avoid the while loop or?

Reply:
Yes we just access the next cell instead of reading every line until we reach a cell.

Reply:
Cool :+1: